Detections/ThreatIntelligenceIndicator/IPEntity_WireData.yaml

```diff
@@ -1,4 +1,4 @@
-id: a50766a7-0674-4ccb-8845-15dc55a80ba1
+id: a50666a7-0674-4ccb-8845-15dc55a80ba1
```

Comment:
Do you realy want to change this id, or you chnage it just for the tests?

Reply:
just for tests, ignore it for mow